akka-http2-support/src/test/scala/akka/http/impl/engine/http2/H2SpecIntegrationSpec.scala

```diff
@@ -96,6 +96,7 @@ class H2SpecIntegrationSpec extends AkkaSpec(
       "4.3",
       "5.1",
       "5.1.1",
+      "5.1.2",
```

Comment:
so h2spec expects us to produce a PROTOCOL_ERROR but we produce a REFUSED_STREAM, which is also allowed by the spec. Too bad but :+1:

Reply:
No no. The problem is `h2spec` tries to create too many connections but sends `HEADERS` frames with the `END_STREAM` flag so we immediately close the streams not triggering the limit. 

I don't know what type of error is expected in `h2spec` when too many streams are open.

Reply:
Ah, right, I misread its outcome (and didn't remember the particulars of our previous discussion on this). I think it accepts all 4 variations (goaway/rst x protocol_error/refused).